WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -444,4 +444,3 @@ private extension Date {
                                                from: self)
     }
 }
-
```

Comment:
linter yelled at me about this line, even though I haven't changed anything here???? so i just removed the whitespace to appease it.

Reply:
You merged `develop` didn't you? 😄 (I had the same problem.)